releases/v2.0.1.md

```diff
@@ -0,0 +1,107 @@
+- Relaxed the limitation on using [`BACKUP`](../v2.0/backup.html) in a mixed version cluster. [#24515][#24515]
+
+### SQL Language Changes
+
+- Improved the error message returned on object creation when no current database is set or only invalid schemas are in the `search_path`. [#24812][#24812]
+- The `current_schema()` and `current_schemas()` [built-in functions](../v2.0/functions-and-operators.html) now only consider valid schemas, like PostgreSQL does. [#24758][#24758]
+- The experimental SQL features `SHOW TESTING_RANGES` and `ALTER ... TESTING_RELOCATE` have been renamed `SHOW EXPERIMENTAL_RANGES` and `ALTER ... EXPERIMENTAL_RELOCATE`. [#24699][#24699]
+- `ROLE`, `VIRTUAL`, and `WORK` are no longer reserved keywords and can again be used as unrestricted names. [#24665][#24665] [#24549][#24549]
+
+### Command-Line Changes
+
+- When [`cockroach gen haproxy`](../v2.0/generate-cockroachdb-resources.html) is run, if an `haproxy.cfg` file already exists in the current directory, it now gets fully overwritten instead of potentially resulting in an unusable config. [#24336][#24336]
+
+### Bug Fixes
+
+- Fixed a bug when using fractional units (e.g., `0.5GiB`) for the `--cache` and `--sql-max-memory` flags of [`cockroach start`](../v2.0/start-a-node.html). [#24388][#24388]
+- Fixed the handling of role membership lookups within transactions. [#24334][#24334]
+- Fixed a bug causing some lookup join queries to report incorrect type errors. [#24825][#24825]
+- `ALTER INDEX ... RENAME` can now be used on the primary index. [#24777][#24777]
+- Fixed a panic involving [inverted index](../v2.0/inverted-indexes.html) queries using the `->` operator. [#24596][#24596]
+- Fix a panic involving [inverted index](../v2.0/inverted-indexes.html) queries over `NULL`. [#24566][#24566]
+- Fixed a bug preventing [inverted index](../v2.0/inverted-indexes.html) queries that have a root with a single entry or element but multiple children overall. [#24376][#24376]
+- [`JSONB`](../v2.0/jsonb.html) values can now be cast to [`STRING`](../v2.0/string.html) values. [#24553][#24553]
+- Prevented planning DistSQL errors against unavailable nodes. [#23916][#23916]
+- Fixed a panic caused by a `WHERE` condition that requires a column to equal a specific value and at the same time equal another column. [#24517][#24517]
+- Fixed a panic caused by passing a `Name` type to `has_database_privilege()`. [#24270][#24270]
+- Fixed a bug causing index backfills to fail in a loop after exceeding the GC TTL of their source table. [#24427][#24427]
+- On-disk checksums are now correctly generated during [`IMPORT`](../v2.0/import.html). If there is existing data created by `IMPORT` that cannot be recreated, use [`cockroach dump`](../v2.0/sql-dump.html) to rewrite any affected tables. [#24221][#24221]
```

Comment:
This change can be removed. It was previously fixed in another way and there's no reason to tell users about it.

Reply:
Done. Was it announced in a previous 2.0 beta release? Couldn't find it referenced.

Reply:
The bug was fixed in two separate ways, one very deep in the KV layer, which is why it doesn't appear obviously. This one just landed second.